imkar/utils/utils.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pyfar as pf
+
+
+def paris_formula(coefficients, incident_directions):
```

Comment:
The name of the function and module suggests that it is public. In this case should we add it to the docs as well?